nomos-da/network/subnetworks-assignations/src/versions/v1.rs

```diff
@@ -42,7 +42,7 @@ impl FillFromNodeList {
 }
 
 impl MembershipHandler for FillFromNodeList {
-    type NetworkId = u32;
+    type NetworkId = u16;
```

Comment:
Don't we have a `SubnetworkId` type already for this? or does it mess circular dependencies?

Reply:
Exactly.
I'll do the middle ground and create a `SubnetworkId = u16` as we did for messages.

Reply:
Or maybe is a hint that this implementation could be generic 🤔 
I actually think it is ok to have the `u16` plain here. As it is an specific implementation for whatever indexes that are `u16`, doesn't matter what name you give it. I think that was my reasoning to use the previous (`u32`) type anyway. So dismiss this comments.

Reply:
Shall I revert the crate-private `SubnetworkId = u16` I did?

Reply:
> Shall I revert the crate-private `SubnetworkId = u16` I did?

Yeah, I think so. Sorry for the misslead.

Reply:
No worries :)